### PR TITLE
PLF-6659: eXoScriptingConsole allows to run source code at server sid…

### PR DESCRIPTION
**Pull request by azaoui**
…e(drop eXoScriptingConsole)

### CODE REVIEW

File: extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml [hunk 1]
```diff
@@ -161,38 +161,6 @@
               </field>
             </object>
           </object-param><object-param>
-            <name>development</name>
```
Comment (azaoui) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml]: Is there any impact for existing customers ?
We think that this gadget is not used so it is not a problem to remove it, but is there any impact on the application registry or on the portlets catalog (when editing a page) for customers who have this gadget declared ? Did you do the test with a 4.3 migrated dataset ?

Comment (azaoui) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml]: @thomasdelhomenie 
The gadget xml are stored in the jcr(portal-system ws)
I tested this case : install the gadget into some page then migrate by removing the app from the server:
the portlet is displayed but of course not working :) as the service was removed, so the end user will understand that the service no long exist so should remove it

Comment (azaoui) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml]: OK, and what about the portlet/gadget catalog ? Is it still usable after the migration ?

Comment (azaoui) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml]: @thomasdelhomenie 
yes as the catalog refer to the stored app registry

Comment (thomasdelhomenie) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml]: yes, that's why I am asking, it refers to the stored app registry which can contain the gadget we dropped, so we have to be sure it still works in this case.

Comment (azaoui) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml]: yes still working as the catalog will display just the category and the name of the gadget things that already stored in database , what we remove is the portlet logic and service , but for the content and the category will remain into database ensuring the app displaying until the user drop manually, 
Other approach is to implement an upgrade plugin to remove the app from the application registry and the pages , but I think it's not a good pactise as the upgrade will be transparent 
and will it delete content and pages without informing the platform users, so it's better to keep content even not working

Comment (thomasdelhomenie) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml]: OK, juste wanted to be sure the catalog does not check for gadget existence.

**Comment by thomasdelhomenie** [comment]
LGTM

**Pull request closed without merging**